.github/workflows/haskell.yml

```diff
@@ -35,6 +35,10 @@ jobs:
           - os: macos-latest
             cabal: "3.10.2.1"
             ghc: "9.6.4"
+        sys:
```

Comment:
`os` in line 31 is redundant now. It will probably need the rest of the changes from https://github.com/IntersectMBO/cardano-api/pull/504/files#diff-c09cc8dc973415b1221308559b96942a1c4b872e690bf360fc22d9b42822d519 too. If it's too big of a change for this PR, this can be done in a follow up PR.

Reply:
Will be done in a later PR.